CONTRIBUTING.md

```diff
@@ -150,6 +150,97 @@ cd test/manual/k8s_crds
 ```
 This test currently does not run as part of the test suite.
 
+## Profiling
+Profiling can be used to monitor the impact of Secretless on CPU and Memory consumption. Currently, Secretless supports two types- CPU and Memory.
+
+**Prerequisites:**
+- **Homebrew**
```

Comment:
this is really only a requirement if you're installing from a Mac. I think it would be better to delete this line and give links to the install pages for graphviz and some page that lists how to install pg clients on different operating systems

Reply:
unresolving this convo bc I don't see any related changes - can you please point me to them if I am missing something?

actually, now that I look you also tell to build by calling `./bin/build_darwin`, which is also mac-specific

Reply:
i fixed it on my end and didn't push the changes hehe whoops

Reply:
should be all fixed now